src/modules/terraform.rs

```diff
@@ -25,6 +27,8 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     }
 
     let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+        let terraform_version = Lazy::new(|| context.exec_cmds_return_first(config.commands));
```

Comment:
Is there a reason you are using `Lazy` instead of putting `let terraform_version = context.exec_cmds_return_first(config.commands)` into the `match`?